easydigitaldownloads/easy-digital-downloads
# Update metabox forms, inputs

Update metabox form fields for semantic and accessible markup.

Fixes #7742

To test:
Check out issue/7742 and edit or create a new download.
- All EDD metabox fields should now be accessible via keyboard navigation (less download product fields which use the Chosen script).
- Clicking on a label should bring focus to the related input fields, even in cloned repeatable rows.
- Using dev tools, you should be able to find a related `<label>` (`for` attribute) for every `<input>` (`id` attribute).
- Using an accessibility checking tool such as axe by Deque, the EDD forms should validate (running on release/3.0 will show errors with missing labels and duplicate IDs).
